packages/mobile/src/geth/geth.ts

```diff
@@ -20,6 +20,8 @@ export const FailedToFetchGenesisBlockError = new Error(
   'Failed to fetch genesis block from Google storage'
 )
 
+export const PROVIDER_CONNECTION_ERROR = "connection error: couldn't connect to node"
```

Comment:
this is different from the previous `BLOCK_CHAIN_CORRUPTION_ERROR`?

Reply:
Just a shorter but still unique subset in case the error message gets formatted slightly differently